api/api.proto

```diff
@@ -1573,6 +1583,139 @@ service OpenStorageCloudBackup {
     }
 }
 
+// START: OpenStoragePolicy Service
+
+// OpenStoragePolicy service provides methods to manager storage policies.
```

Comment:
This needs to be on top of the service for the doc generator to work. See other services